hostsensorutils/hostsensorgetfrompod.go

```diff
@@ -104,7 +105,41 @@ func (hsh *HostSensorHandler) GetLinuxSecurityHardeningStatus() ([]hostsensor.Ho
 // return list of KubeletCommandLine
 func (hsh *HostSensorHandler) GetKubeletCommandLine() ([]hostsensor.HostSensorDataEnvelope, error) {
 	// loop over pods and port-forward it to each of them
-	return hsh.sendAllPodsHTTPGETRequest("/kubeletCommandLine", "KubeletCommandLine")
+	resps, err := hsh.sendAllPodsHTTPGETRequest("/kubeletCommandLine", "KubeletCommandLine")
+	if err != nil {
+		return resps, err
+	}
+	for resp := range resps {
+		data := makeCmdLineMap(string(resps[resp].Data))
+		resBytesMarshal, err := json.Marshal(data)
+		// TODO catch error
+		if err == nil {
+			resps[resp].Data = json.RawMessage(resBytesMarshal)
+		}
+	}
+
+	return resps, nil
+
+}
+
+func makeCmdLineMap(fullCommand string) map[string]interface{} {
+
+	commands := strings.Split(fullCommand, " ")
+	if len(commands) < 1 {
+		return nil
+	}
+	command := commands[0]
+	var cmdMap = make(map[string]interface{})
+	cmdMap["command"] = command
+	for _, cmd := range commands {
+		splitted := strings.Split(cmd, "=")
```

Comment:
1. What if there is a boolean flag? (e.g. `--submit`)
2. What if the flag has a single `-`? (e.g. `-t`)